### PR TITLE
Use k8s-1.20 for vm-import-operator job

### PR DESCRIPTION
**Pull request by mansam**
@dhiller 

Signed-off-by: Sam Lucidi <slucidi@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/vm-import-operator/OWNERS [hunk 1]
```diff
@@ -5,8 +5,10 @@ filters:
       - machacekondra
       - masayag
       - pkliczewski
+      - mansam
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/vm-import-operator/OWNERS:8]: Please note that if you had an OWNERS file in kubevirt/vm-import-operator, this OWNERS file here would automatically be in sync with the OWNERS from kubevirt/vm-import-operator.

**Review by dhiller — APPROVED** [review]
I have commented on the other PR already, these changes look good.

/approve

File: github/ci/prow-deploy/files/jobs/kubevirt/vm-import-operator/OWNERS (hunk 1 above)
Comment (mansam) [reply github/ci/prow-deploy/files/jobs/kubevirt/vm-import-operator/OWNERS:8]: Ah, I see, thanks. I'll remedy that in `kubevirt/vm-import-operator`.

**Comment by mansam** [comment]
@dhiller The test suite is running correctly now; the remaining test failures are just vm-import-operator issues to be fixed in that repo. Can this be merged?

**Pull request merged**